### PR TITLE
Improve Menu of the Servo website

### PR DESCRIPTION
**Pull request by atbrakhi**
Our menu right now is a bit confusing. In this PR I am proposing some changes to it:

- [x] Remove `Get Involved` page as we already have contact us on home page. 
- [x] Add the information we had on `Get Involved` page in `Contribute` page. So that we don't have to navigate some where else to find links to get involved
- [x] Remove `Sponsorship` from Menu
- [x] Add `Sponsorship` as subpages in `About`
- [x] Add  `Acknowledgements` as subpages in `About`
- [x] Reshuffle the order of Menu
- [x] Only navbar-item should have links. Example: `About` is not a navbar-iteam but `About Us` is. Hence, only `About us` has a page link `About` doesn't.

Attaching some screenshot:
image
image

### CODE REVIEW

File: _data/menu.json [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
-  { "title": "Sponsorship", "url": "/sponsorship/", "subpages": [
-    { "title": "Acknowledgements", "url": "/acknowledgements/" }
-  ] },
-  { "title": "Blog", "url": "/blog/" }
+  { "title": "Blog", "url": "/blog/" },
+  {
+    "title": "About",
+    "url": "#",
+    "subpages": [
+      { "title": "About Us", "url": "/about/" },
```
Comment (delan) [inline _data/menu.json:17]: i think “About Servo” would make more sense here, wdyt?

Comment (atbrakhi) [reply _data/menu.json]: indeed. thanks for the review :)
Fixed in [35ece43](https://github.com/servo/servo.org/pull/107/commits/35ece439131fbdfb7e9b23351aed30ed26be3c58)

**Comment by matlu** [comment]
Thanks.

I still believe it would be valuable to also have direct links to:

- Roadmap
- WPT pass rate
- Servo Presentations

Which all 3 today are "hidden" under "About" in https://servo.org/about/

Maybe they can be intended on level under "About"  in the drop down menu?

I agree that Governance, Sponsorship and Acknowledgements can also be under About.

**Comment by matlu** [comment]
> Thanks.
> 
> I still believe it would be valuable to also have direct links to:
> 
> * Roadmap
> * WPT pass rate
> * Servo Presentations
> 
> Which all 3 today are "hidden" under "About" in https://servo.org/about/
> 
> Maybe they can be intended on level under "About" in the drop down menu?
> 
> I agree that Governance, Sponsorship and Acknowledgements can also be under About.

I also suggest that our greatly improved Wiki (https://github.com/servo/servo/wiki) can get a new top-level entry on Servo.org or also add it to the growing list of options under "About"

**Comment by matlu** [comment]
> > Thanks.
> > I still believe it would be valuable to also have direct links to:
> > 
> > * Roadmap
> > * WPT pass rate
> > * Servo Presentations
> > 
> > Which all 3 today are "hidden" under "About" in https://servo.org/about/
> > Maybe they can be intended on level under "About" in the drop down menu?
> > I agree that Governance, Sponsorship and Acknowledgements can also be under About.
> 
> I also suggest that our greatly improved Wiki (https://github.com/servo/servo/wiki) can get a new top-level entry on Servo.org or also add it to the growing list of options under "About"



> > Thanks.
> > I still believe it would be valuable to also have direct links to:
> > 
> > * Roadmap
> > * WPT pass rate
> > * Servo Presentations
> > 
> > Which all 3 today are "hidden" under "About" in https://servo.org/about/
> > Maybe they can be intended on level under "About" in the drop down menu?
> > I agree that Governance, Sponsorship and Acknowledgements can also be under About.
> 
> I also suggest that our greatly improved Wiki (https://github.com/servo/servo/wiki) can get a new top-level entry on Servo.org or also add it to the growing list of options under "About"

One more thing. As we do this change and also soon have our very important month of Outreach in front of us (September).
Can we also add Demos aka Servo Experiments to the top level?

https://wpewebkit.org/about/ uses the more inspiring name:
_See WPE In Action…_
for their demos.
So e.g. add "See Servo In Action" as  new entry under About would also work.

**Comment by atbrakhi** [comment]
@matlu thanks for the feedback. For initials I have included the link to Servo Demo, Wiki and WPT test pages in menu.

I will look further for some inspiration on UX and see if we can integrate it on servo website itself and open a separate PR for that. I propose, we keep this pull request limited to improving menu of servo website.

@delan let me know if you agree with the changes.

File: _data/menu.json [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
-    { "title": "Contributing", "url": "/contributing/" },
-    { "title": "Documentation", "url": "/documentation/" },
-    { "title": "Code of Conduct", "url": "/coc/" }
-  ] },
-  { "title": "Sponsorship", "url": "/sponsorship/", "subpages": [
-    { "title": "Acknowledgements", "url": "/acknowledgements/" }
-  ] },
-  { "title": "Blog", "url": "/blog/" }
+  { "title": "Blog", "url": "/blog/" },
+  { "title": "Demo", "url": "https://demo.servo.org/" },
```
Comment (delan) [inline _data/menu.json:14]: Demo → Demos

File: _data/menu.json [hunk 3]
```diff
@@ -1,15 +1,25 @@
 [
-  { "title": "About", "url": "/about/", "subpages": [
-    { "title": "Governance", "url": "/governance/" }
-  ] },
+  {
+    "title": "Get Involved",
+    "url": "#",
+    "subpages": [
+      { "title": "Contributing", "url": "/contributing/" },
+      { "title": "Servo Wiki", "url": "https://github.com/servo/servo/wiki" },
```
Comment (delan) [inline _data/menu.json:7]: Servo Wiki → Wiki

File: _data/menu.json [hunk 4]
```diff
@@ -1,15 +1,25 @@
...
+      { "title": "Servo WPT", "url": "https://wpt.servo.org/" },
```
Comment (delan) [inline _data/menu.json:8]: Servo WPT → WPT pass rates (use sentence case)

File: contributing.md [hunk 5]
```diff
@@ -11,6 +11,13 @@ If you want to start contributing to Servo, check out [`CONTRIBUTING.md`](https:
 
 You can search for and/or report any issues you run into on the [GitHub issue tracker](https://github.com/servo/servo/issues).
 
+## Questions? Contact us!
+
+* Ask questions in [Servo Zulip chat](https://servo.zulipchat.com/)
+* Start a discussion on [GitHub](https://github.com/servo/servo/discussions)
+* Tag us on social media [Mastodon](https://floss.social/@servo) and [Twitter](https://twitter.com/ServoDev) accounts
```
Comment (delan) [inline contributing.md:18]: on social media → on

Comment (delan) [inline contributing.md:18]: remove “social media” and “accounts”

**Comment by matlu** [comment]
@atbrakhi @delan
FYI @mrego 

Great first improvements!

Suggested additional fixes:

1. Can we integrate demos so it is using the servo.org theme and looks more like how: https://wpewebkit.org/about/ have their demos integrated. Should still have its own top level menu item but a look a bit more aligned would be great. Maybe this can be done at the same time you add WebGL demos like Fishtank and WebGPU demos once we have them confirmed working.
2. What is still missing is the key information under "About" which is:

**Roadmap** and **Presentations**

Both of these are key information when you try to sell in/pitch Servo project towards potential new partners.  So can we find a way to surface these to headings in the menu structure?

When you click on the "About" in for example "Google Chrome Browser you only find the boring stuff so most users including myself does not expect "About" to include the great stuff like Roadmap and Presentations.

Great if you can find a solution for this and implement before end of August as September is a busy month for outreach.


Thanks!

PS. At some stage in Q4 we should hopefully also be able to add "Showcase" or similar like what you can see on Wonderland Engine web site: https://wonderlandengine.com/

image

**Pull request merged**